debezium-core/src/main/java/io/debezium/pipeline/EventDispatcher.java

```diff
@@ -293,6 +293,11 @@ public void changeRecord(P partition,
                             streamingReceiver.changeRecord(partition, schema, operation, key, value, offset, headers);
                         }
                     }
+
+                    private boolean isASignalEventToProcess(T dataCollectionId, Operation operation) {
+                        return operation == Operation.CREATE &&
```

Comment:
Should not this support `DELETE` as well?

Reply:
I did myself the same question. Enabling `DELETE` will avoid to directly close the window because it will be then processed when the delete happens instead of close insert. I was going to this path but then I thought that this will have a side effect. If the user delete something on the signal table the event will be processed again. So i preferred to use this approach.

Reply:
@mfvitale One question. What if `DELETE` is executed but the connector crashesh before the offsets are saved. What will happen then?

Reply:
Having a look to the code again seems that since the commit is after the call to the `closeWindow` we should be fine also in case of crash before the call to it. Am I right?

Reply:
ok, I tried to come up with multiple scenarios of when the crash can happen it seems the soultion is resilient to it.